packages/core/src/global/grid.scss

```
@@ -0,0 +1,70 @@
+.tds-grid-container {
+  display: flex;
+  justify-content: center;
+}
+
+.tds-grid-fixed,
+.tds-grid-fluid {
+  display: grid;
+  width: calc(100% - 32px);
+  padding: 0 16px;
+  grid-template-columns: repeat(6, 1fr);
+  grid-template-rows: repeat(2, 1fr);
+  grid-gap: 16px;
+}
```

```suggestion
.tds-grid-fixed,
.tds-grid-fluid {
  display: grid;
  width: calc(100% - 32px);
  padding: 0 16px;
  grid-template-columns: repeat(6, 1fr);
  grid-gap: 16px;
}
```

Line related to the number of the rows should not be part of code Tegel ships. It is up to user to specify on their side. However, you will need it for demo purposes in Storybook, so please add it under story demo code:
`  grid-template-rows: repeat(2, 1fr);`